src/ray/core_worker/task_event_buffer.h

```diff
@@ -173,13 +177,23 @@ enum TaskEventBufferCounter {
 ///
 /// Dropping of task events
 /// ========================
-/// Task events will be lost in the below cases for now:
+/// Task events from task attempts will be lost in the below cases for now:
 ///   1. If any of the gRPC call failed, the task events will be dropped and warnings
 ///   logged. This is probably fine since this usually indicated a much worse issue.
 ///
 ///   2. More than `RAY_task_events_max_buffer_size` tasks have been stored
-///   in the buffer, any new task events will be dropped. In this case, the number of
-///   dropped task events will also be included in the next flush to surface this.
+///   in the buffer, oldest events in the buffer will be dropped. In this case, the task
+///   attempts info will also be included in subsequent flush to GCS.
+///
+/// For profiling events:
+///   - If the number of profiling events for a task attempt exceeds the limit specified
+///   by `RAY_task_max_profile_events_per_task`, any new profiling events will be dropped.
```

Comment:
Is it the correct name? I couldn't find it from the config

Reply:
task_max_profile_events_per_task